docs/pages/api.rst

```diff
@@ -1853,7 +1799,8 @@ Signature: ``Collection::tail();``
 tails
 ~~~~~
 
-Returns the list of initial segments of its argument list, shortest last.
+Returns the list of initial segments of the collection, shortest last.
+Similar to applying ``tail`` successively and collecting all results in one list.
```

Comment:
once I read your blog and realised the Haskell influence many of these methods started to make more sense 😄 . The initial definition was pretty much like the Haskell one hehe, but in our case `tails` doesn't take arguments

Reply:
Ahaha you went down into the rabbit hole :) congrats! Haskell is so beautiful that I wanted to provide the same kind in PHP.

However, `[tails](https://hackage.haskell.org/package/base-4.15.0.0/docs/Data-List.html#v:tails)` in Haskell doesn't take arguments.

Reply:
I like it, though my experience with it is pretty much limited to reading this [book](http://learnyouahaskell.com/) and trying out some exercises 😄

Reply:
Same here and also this one, which is actually better: https://www.manning.com/books/get-programming-with-haskell

Reply:
thanks, I'll check it out!